lib/src/data/model/tmdb_item_model.dart

```diff
@@ -0,0 +1,21 @@
+import '../../../src/domain/entity/movie.dart';
+
+class TMDBItemModel {
```

Comment:
This is not a model, is a response, instead to TMDBItemModel you can replace to ResponseModel

Reply:
And also check the names, in the future if we want to change the api instead to use TMDB we want to use Pokemon and for some reason the models are the same, do you think that the name is correct?

Reply:
fixed